package.json

```diff
@@ -30,9 +30,9 @@
     "clean:modules": "del ./**/node_modules",
     "clean": "del ./dist && yarn run schematics:clean",
     "start": "yarn angular:dev:start",
-    "build": "npm-run-all build:ci build:website -p build:storybook:core build:storybook:angular angular:dev:build",
```

Comment:
Missing the `build:storybook:angular` script here.

Reply:
I removed it temporarily because it was failing from Storybook not being updated to work with Angular 11, and we aren't currently utilizing the Angular storybook app.